packages/react/src/components/UIShell/HeaderPanel.js

```diff
@@ -13,15 +13,14 @@ import { AriaLabelPropType } from '../../prop-types/AriaPropTypes';
 
 const { prefix } = settings;
 
-const HeaderPanel = React.forwardRef(function HeaderPanel(props) {
-  const {
-    'aria-label': ariaLabel,
-    'aria-labelledby': ariaLabelledBy,
-    children,
-    className: customClassName,
-    expanded,
-  } = props;
-
+const HeaderPanel = ({
```

Comment:
Any reason we stopped do-ing `forwardRef()` here? I didn't think `forwardRef()` itself caused #2964?